### ISSUE TITLE
[FEAT] - why don't we make website along side the API using the data from the API?

### ISSUE THREAD

**Issue by dbsaw** [MizouziE/energy-prices-api#40]

**Comment by MizouziE** [comment]
I agree that it's a great idea to have, but what if we go one step further and encourage others to make web apps that consume the data and we use the new front/landing page to keep an index of them?

API consumption is a nice mini project for a SPA and the possibilities are endless, so I'd be very interested to see the different ways people make use of the API. 

I have the API registered on RapidAPI and that can handle API keys etc.

Perhaps we can add another document giving basic steps to proper use of the data from the API itself? Of course for this I will need to:
- Improve documentation on the [Rapid API Hub instance](https://rapidapi.com/sanglin-TlEqlfGPc/api/energy-price-news/)
- Include steps in generating an API key
- Rapid API has nice code snippets for use with different languages and libraries too which opens it up to a bigger audience

One other idea would be to create a package that makes for easy widget style implementation of the API. I will look into moving this project over to an organisation to try and keep it all in one place.

**Comment by dbsaw** [comment]
Yes even better, I haven't worked with something like that before so it's a new journey for me and I'm sure I will have fun learning about it and maybe helping you implement it.

**Comment by MizouziE** [comment]
> Yes even better, I haven't worked with something like that before so it's a new journey for me and I'm sure I will have fun learning about it and maybe helping you implement it.

If you build anything with it be sure to raise an issue/make a PR to include a link to it on the homepage! And also ask any questions if you have any on here or find my contact details on my profile. 👊

**Comment by TAKANOME-DEV** [comment]
I think we can start brainstorming about this :hammer_and_wrench:

**Comment by MizouziE** [comment]
@TAKANOME-DEV Now that it's in an organization, we can make more repos to have different types of front end approaches. 

There is one example frontend app that is linked from the homepage already, but it's the only one so far.

**Comment by TAKANOME-DEV** [comment]
Oh I didn't pay attention to that, I will check it out.
Yeah you can create more repos for examples 🚀
